sql/core/src/test/resources/sql-tests/inputs/datetime.sql

```diff
@@ -160,3 +156,83 @@ select from_json('{"time":"26/October/2015"}', 'time Timestamp', map('timestampF
 select from_json('{"date":"26/October/2015"}', 'date Date', map('dateFormat', 'dd/MMMMM/yyyy'));
 select from_csv('26/October/2015', 'time Timestamp', map('timestampFormat', 'dd/MMMMM/yyyy'));
 select from_csv('26/October/2015', 'date Date', map('dateFormat', 'dd/MMMMM/yyyy'));
+
+select date_format(null, null);
```

Comment:
These tests were added because datetime function end-2-end tests were missing so far

Reply:
we can add `from_utc_timestamp` and `to_utc_timestamp` here later too